rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
+
+  def initialize(date, person, book)
+    @date = date
+
+    @person = person
+    person.rentals << self
```

Comment:
- [ ] What you have in your Person class is `rental`, not `rentals` so kindly fix one of the attributes for consistency and to avoid errors.

Reply:
@TedLivist Thanks for your feedback, I will make the code change.